bugzoo/mgr/source.py

```diff
@@ -168,17 +168,11 @@ def __parse_blueprint(self, source: Source, fn: str, d: dict) -> BuildInstructio
                                  source.name)
 
     def __parse_bug(self, source: Source, fn: str, d: dict) -> Bug:
```

Comment:
is `fn` used anywhere?

Reply:
Yes and no. In general, we use `fn` to figure out the directory that contains the bug manifest. We then use that directory to determine the absolute path to the `Dockerfile` (and build context) specified in the manifest. However, we only need that logic for blueprints, and not necessarily for bugs.  For the sake of consistency, I think it's worth keeping `fn` in the `__parse_bug` header for now, even if it isn't used directly.